assets/tests/components/notifications.test.tsx

```diff
@@ -155,6 +159,7 @@ describe("NotificationCard", () => {
         notification={notification}
         remove={jest.fn()}
         currentTime={now()}
+        fetchCurrentVehicle={() => null}
```

Comment:
`jest.fn()` is the standard way to make empty functions for this (and then you can have a test that asserts that it's called on click).